io/src/librealsense/librealsense_device_manager.cpp

```diff
@@ -120,7 +119,7 @@ void
 pcl::io::librealsense::LibRealSenseDeviceManager::releaseDevice (const std::string& sn)
 {
   boost::mutex::scoped_lock lock (mutex_);
-  pcl::io::librealsense::LibRealSenseDeviceManager::stopDevice (sn);
+  if (!thread_close_flag_[sn]) pcl::io::librealsense::LibRealSenseDeviceManager::stopDevice (sn);
```

Comment:
I think releaseDevice means deviceManager give up the control of the giving device, so it is necessary to check if the device is running and stop the device in case the other grabber want to use this device again.